lib/pages/new_group/new_group.dart

```diff
@@ -168,6 +168,21 @@ class NewGroupController extends State<NewGroup>
     );
   }
 
+  void createNewGroup({ required String? uriAvatar }) {
```

Comment:
```suggestion
  void createNewGroup({String? uriAvatar}) {
```

Reply:
Why not keep it is Uri? Very clearly, Uri is Uri, not a String. We have a strong type, why need to remove it?

Reply:
done